Sources/SwiftProtobuf/UnknownStorage.swift

```diff
@@ -25,11 +25,15 @@ public struct UnknownStorage: Equatable {
 
   public init() {}
 
+  public init(data: Data) {
```

Comment:
Needed?  I was going to say now that append is internal, the only thing that can get into the unknowns should come from within the library, so shouldn't that ensure the data is always valid and there be no need to throw?